### PR TITLE
RHDEVDOCS-4274: Added the bug#2069258 in the bug list for OCP 4.9.31

### PR DESCRIPTION
**Pull request by shipsing**
Aligned team: Dev Tools

Purpose: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-4274

OCP version this PR applies to: 4.9 Release Notes only

Link to docs preview: [Download to see preview in your browser](https://drive.google.com/file/d/1f3GHguRy6H5R8VA7havdwBrmQ2WR7u4x/view?usp=sharing)

SME acknowledged in Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2069258  @rottencandy

QE approval needed: @sanketpathak 

Peer review completed: @jc-berger 

**Change Management** 
Need an ACK from Eng: Provided by @rottencandy 
Need an ACK from PM: Provided by @serenamarie125 
Need an ACK from Product experience:  Provided by @RickJWagner 
Need an ACK from QE: Provided by @sanketpathak 
Need an ACK from DPM: Provided by @cbremble 
Need an ACK from CS: Provided by @TanyaShearonRH

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc
Comment (jc-berger) [inline release_notes/ocp-4-9-release-notes.adoc:2818]: Rolfe also looked at this PR, and he had some good suggestions. The IBM SG wants us to use `would` as a hypothesis, or an unfulfilled condition. 

What do you think of Rolfe's suggested rewrites for the following?: 

 `if users provided a secret` instead of "when users would provide a secret" 

`which helps to detect OR which helps detect` instead of "which helps detecting"

Comment (shipsing) [reply release_notes/ocp-4-9-release-notes.adoc]: Thank you @jc-berger for this suggestion. I have incorporated the same. Please verify and confirm.

**Comment by shipsing** [comment]
@serenamarie125 @RickJWagner, Could you please provide an ack for this PR so that we can proceed with merge?
If you have any comments, you can also share the same.
Thanks

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by shipsing** [comment]
@serenamarie125, Could you please provide an ack for this PR so that we can proceed with merge? Rest of the approvals are done.

Thanks

**Comment by serenamarie125** [comment]
@shipsing I approve, was trying to use the same labeling technique as Rick & Claire, but doesn't seem to work.

**Review by serenamarie125 — APPROVED** [review]
Approved! lgtm

File: release_notes/ocp-4-9-release-notes.adoc
Comment (bburt-rh) [inline release_notes/ocp-4-9-release-notes.adoc:2818]:
```suggestion
* Before this update, if users provided a secret for a private repository in the *Import from Git* form, the secret was not decoded. As a result, the *Import from Git* form failed to detect the correct import strategy and builder image for the repository. With this update, the *Import from Git* form decodes the secret before its use, which helps detect the correct import strategy and builder image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069258[*BZ#2069258*])
```
Please fix the typo (extra space)

**Comment by bburt-rh** [comment]
@serenamarie125 Are you the QE approver for this? or the PM? It's not clear in the PR description. It needs QE approval before I can merge it. Thanks!

**Comment by cbremble** [comment]
@sanketpathak could you please provide a QE approval for this change to be made to the docs? This is updating the 4.9.31 RN with the doc text you acked in the BZ here: https://bugzilla.redhat.com/show_bug.cgi?id=2069258. Thank you!

File: release_notes/ocp-4-9-release-notes.adoc
Comment (bburt-rh) [inline release_notes/ocp-4-9-release-notes.adoc:2818]:
```suggestion
* Before this update, if users provided a secret for a private repository in the *Import from Git* form, the secret was not decoded. As a result, the *Import from Git* form failed to detect the correct import strategy and builder image for the repository. With this update, the *Import from Git* form decodes the secret before its use, which helps detect the correct import strategy and builder image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069258[*BZ#2069258*])
```
Remove the extra space between the 2nd and 3rd sentences

Comment (shipsing) [reply release_notes/ocp-4-9-release-notes.adoc]: Incorporated

**Comment by shipsing** [comment]
Hello @sanketpathak @rottencandy, Could you please provide your approvals for this PR? Thanks

**Review by rottencandy — APPROVED** [review]
lgtm, thanks.

**Comment by sanketpathak** [comment]
LGTM, thanks

**Pull request merged**
